libs/ui-lib/lib/ocm/components/clusterConfiguration/CustomManifestCheckbox.tsx

```diff
@@ -46,30 +46,20 @@ const CustomManifestCheckbox = ({ clusterId, isDisabled }: CustomManifestCheckbo
 
   const onChanged = React.useCallback(
     (checked: boolean) => {
-      if (!checked) {
-        //For new clusters is not necessary to show the delete modal
-        if (clusterId) setDeleteCustomManifestsOpen(true);
-        else {
-          cleanCustomManifests();
-        }
-      } else {
-        setValue(checked);
-        setManifestsRemoved(false);
-        clusterWizardContext.setAddCustomManifests(checked);
-      }
+      clusterWizardContext.setCustomManifestsStep(checked);
```

Comment:
PTAL at line [77](https://github.com/openshift-assisted/assisted-installer-ui/pull/2374/commits/ba99039db3beff1789eccaa9949ac6233b374800#diff-a4b0fa62e02584dcfa2e8fedae16a7e2f632bf1dec4bc17ea797be076e8a0403R77) despite this function is inside a useCallback, there we are creating another arrow function.